auto-generated/vector-crypto/api-testing/vaeskf1.c

```diff
@@ -0,0 +1,26 @@
+#include <stdint.h>
+#include <riscv_vector.h>
+
+typedef _Float16 float16_t;
+typedef float float32_t;
+typedef double float64_t;
+vuint32mf2_t test_vaeskf1_vi_u32mf2(vuint32mf2_t vs2, size_t uimm, size_t vl) {
+  return __riscv_vaeskf1_vi_u32mf2(vs2, 0, vl);
```

Comment:
This should use a constant that is meaningful is the spec. 1-14 are the true valid valid values for this instruction.